templates/macros/event.njk

```diff
@@ -6,6 +6,19 @@
   {% endif %}
 {% endmacro %}
 
+{% macro invert(twitterInfo) %}
+  {% if twitterInfo.hashtag %}
+    invert
+  {% endif %}
+{% endmacro %}
+
+{% macro dateInfo(isToday, isThisWeek, twitterInfo) %}
+  {% if isToday %}
+    <div class="date-badge {{invert(twitterInfo)}}">Hoy</div>
```

Comment:
Just to let you know, a pattern I do in other repos is to return all the class names in one method, I've found that is more clear, as in:
```
<div class="{{dateBadgeClassNames(twitterInfo)}}">Hoy</div>
...
```

Reply:
Seems like a good idea :)